File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=len(nums)
        thatnum=0
        for i in range(n):
            if i not in nums:
                thatnum=i
                return thatnum
        if thatnum==0:
            return n

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int[] arr = new int[n + 1];
        Arrays.fill(arr, -1);
        for (int i = 0; i < n; i++) {
            arr[nums[i]] = nums[i];
        }
        for (int i = 0; i < n + 1; i++) {
            if (arr[i] == -1)
                return i;
        }
        return 0;
    }
}